testpilot/frontend/static-src/app/views/retire-page.js

```diff
@@ -0,0 +1,76 @@
+import app from 'ampersand-app';
+import PageView from './page-view';
+
+export default PageView.extend({
+  pageTitle: 'Firefox Test Pilot',
+  pageTitleL10nID: 'pageTitleRetirePage',
+  template: `
+    <div class="blue">
+      <div class="stars"></div>
+      <div class="full-page-wrapper centered">
+        <div class="centered-banner">
+          <div disabled class="loading-pill">
+            <h1 class="emphasis" data-l10n-id="retirePageProgressMessage">Shutting down Test Pilot...</h1>
+            <div style="opacity: 1" class="state-change-inner"></div>
```

Comment:
This is part of the styling stuff that needs a follow up PR. It's not finished here.